canonical-kubernetes/addons/jfrog/steps/01_install_arti/after-deploy

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+
+set -eux
+
+. "$CONJURE_UP_SPELLSDIR/sdk/common.sh"
+. "$CONJURE_UP_SPELLSDIR/$CONJURE_UP_SPELL/addons/common.sh"
+
+export KUBECONFIG=$HOME/.kube/config.$JUJU_MODEL
```

Comment:
Other than kubectl, helm also has reference to KUBECONFIG. Is that going to change as well ?

Reply:
The single standard `$HOME/.kube/config` file is now used so there should be no need to set `KUBECONFIG` anywhere.  PR #180 removed the references from the Helm add-on.  This branch is 10 commits behind master, so it might be worth rebasing or merging those changes in.

Reply:
For reference, these are the (non-merge) commits in master:

```
  6104c5e : 2018-04-18 : George Kraft   : Fix helm kubeconfig issues 
  78e364b : 2018-04-18 : Kevin W Monroe : kubeflow addon should use kubectl from step 2 (and remove unneeded KUBECONFIG) 
  dbb4306 : 2018-04-18 : Kevin W Monroe : remove unnecessary CONJURE_UP_SESSION_ID 
  b4f4818 : 2018-04-18 : Kevin W Monroe : use kubectl from step 2 in subsequent steps 
  aaf7bdb : 2018-04-18 : Kevin W Monroe : remove unnecessary nvidia config in cdk-nvidia (now shares all steps with cdk) 
  b44f7d0 : 2018-04-18 : George Kraft   : Fix helm failure when ~/bin doesn't exist 
  fdb741d : 2018-04-16 : Cory Johns     : Avoid spurious warning when kubectl already installed 
  8a10401 : 2018-04-16 : Cory Johns     : Fix permission denied error in 02_get-kubectl after-deploy 
```